scripts/auto_record.sh

```diff
@@ -31,7 +31,7 @@ echo "Task: $1"
 for (( i=0; i<$2; i++ ))
 do
   echo "Starting episode $i"
-  python3 "$RECORD_EPISODES" --task "$1"
+  python3 "$RECORD_EPISODES" --task "$1" $3 $4
```

Comment:
Put args in double quotes

```suggestion
  python3 "$RECORD_EPISODES" --task "$1" "$3" "$4"
```

Reply:
The python script will complaint about getting `python3 record_episodes.py --task "xxx" "-g"`. But we can remove the "" for $1 to keep the style consistent. What do you think?

Reply:
Sure, that works for me.

Reply:
I looked into this more. It was the optional argument that caused the problem. We are getting `python3 record_episodes.py --task "xxx" "-g" ""`, where the last one is a empty str not defined in the arg parser. Either way, not using the quotes seems to be a good solution.